conans/client/build/msbuild.py

```diff
@@ -87,6 +89,15 @@ def get_command(self, project_file, props_file_path=None, targets=None, upgrade_
                 self._output.warn("***** The configuration %s does not exist in this solution *****" % config)
                 self._output.warn("Use 'platforms' argument to define your architectures")
 
+        if output_binary_log is not None and output_binary_log:
+            msbuild_version = MSBuild.get_version(self._settings)
+            if msbuild_version > "15.3":  # http://msbuildlog.com/
```

Comment:
```suggestion
            if msbuild_version >= "15.3":  # http://msbuildlog.com/
```

I understand that version 15.3 will work.